CI failure in python 3 integration tests:

```
2017-04-13 16:27:48 ERROR! Unexpected Exception: write() argument must be str, not bytes
2017-04-13 16:27:48 the full traceback was:
2017-04-13 16:27:48 
2017-04-13 16:27:48 Traceback (most recent call last):
2017-04-13 16:27:48   File "/root/ansible/bin/ansible-vault", line 109, in <module>
2017-04-13 16:27:48     exit_code = cli.run()
2017-04-13 16:27:48   File "/root/ansible/lib/ansible/cli/vault.py", line 173, in run
2017-04-13 16:27:48     self.execute()
2017-04-13 16:27:48   File "/root/ansible/lib/ansible/cli/__init__.py", line 149, in execute
2017-04-13 16:27:48     fn()
2017-04-13 16:27:48   File "/root/ansible/lib/ansible/cli/vault.py", line 339, in execute_decrypt
2017-04-13 16:27:48     self.editor.decrypt_file(f, output_file=self.options.output_file)
2017-04-13 16:27:48   File "/root/ansible/lib/ansible/parsing/vault/__init__.py", line 447, in decrypt_file
2017-04-13 16:27:48     self.write_data(plaintext, output_file or filename, shred=False)
2017-04-13 16:27:48   File "/root/ansible/lib/ansible/parsing/vault/__init__.py", line 547, in write_data
2017-04-13 16:27:48     sys.stdout.write(b_file_data)
2017-04-13 16:27:48 TypeError: write() argument must be str, not bytes
```

--- a/lib/ansible/parsing/vault/__init__.py
+++ b/lib/ansible/parsing/vault/__init__.py
@@ -544,7 +544,8 @@ class VaultEditor:
         b_file_data = to_bytes(data, errors='strict')
 
         if filename == '-':
-            sys.stdout.write(b_file_data)
+            file_data = to_text(b_file_data, encoding='utf-8', errors='strict', nonstring='strict')
+            sys.stdout.write(file_data)
         else:
             if os.path.isfile(filename):
                 if shred:

